index.html

```diff
@@ -0,0 +1,127 @@
+  <link
+  rel="stylesheet"
+  href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+  integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+  crossorigin="anonymous"
+  /> 
+  <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+  <header>
+    <div class="nav-top">
+      <ul class="nav-menu-top ">
+        <li class="nav-item-top"><i class="fab fa-facebook-f nav-link-top"></i></li>
+        <li class="nav-item-top"><i class="fab fa-twitter nav-link-top"></i></li>         
+        <li class="nav-item-top"><a href="#" class="nav-link-top">English</a></li>
+        <li class="nav-item-top"><a href="#" class="nav-link-top">My Page</a></li>
+        <li class="nav-item-top"><a href="#" class="nav-link-top">Login</a></li>
+      </ul>
+    </div>
+
+    <nav class="navbar">
+      <a href="#" class="logo">AC <span>2022</span></a>
+      <ul class="nav-menu ">
+        <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+        <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+        <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+        <li class="nav-item"><a href="#" class="nav-link">AC Campaign</a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <section id="hero">
+      <h2>"Hello World! We Are Algerian Coders</h2>
+      <h1>البرمجة الجزائرية Algerian Coder 2022</h1>
+      <div><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque a ipsa delectus eum totam perferendis.</p></div>
+      <h3>2022 10.22(thu) ~ 16(fri)</h3>  
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ex, iusto?</p>
+    </section>
+
+    <section id="program">
+      <h2>Main Program</h2>
+      <div class="card-list">
+     
+        <div class="program-card">
+          <ul>
+            <li><img src="images/program-icon/icons-1.svg" alt="Program Icon"></li>
+            <li><h4><b>Lecture</b></h4></li>
+            <li><p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Illum, harum!</p></li>
+          </ul>
+        </div>
+        <div class="program-card">
+          <ul>
+            <li><img src="images/program-icon/icons-4.svg" alt="Program Icon"></li>
+            <li><h4><b>Forum</b></h4></li>
+            <li><p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Illum, harum!</p></li>
+          </ul>
+        </div>
+        <div class="program-card">
+          <ul>
+            <li><img src="images/program-icon/icons-5.svg" alt="Program Icon"></li>
+            <li><h4><b>AC 2022</b></h4></li>
+            <li><p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Illum, harum!</p></li>
+          </ul>
+        </div>
```

Comment:
- [ ] From the design the cards are supposed to be five in number, kindly fix this
              design 👇 
             
![image](https://user-images.githubusercontent.com/66097371/153713760-553f7e2e-5620-455d-94ac-ddd1886d93df.png)